spec/controllers/api/v1/book_suggestion_controller_spec.rb

```diff
@@ -0,0 +1,46 @@
+describe Api::V1::BookSuggestionController, type: :controller do
+  include_context 'Authenticated User'
+  include Devise::Test::ControllerHelpers
+
+  describe 'POST #create' do
+    context 'When creating a BookSuggestion with a valid user' do
+      let!(:book_sugg) { create(:book_suggestion, user: user) }
```

Comment:
Here 2 points:
1. `!` is not useful, if you want we can talk about it by call;
2. Do not create the entity if you will you the `create` end point here. Maybe you should try something like that:
```ruby
let(:book_sugg) { attributes_for(:book_suggestion, user: user) }
```